test/lsp/workspaceSymbol/workspaceSymbol.rec

```diff
@@ -3,4 +3,4 @@ Write: {"jsonrpc":"2.0","id":0,"requestMethod":"initialize","result":{"capabilit
 Read: {"jsonrpc":"2.0","method":"initialized","params":{}}
 Read: {"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"file:///Users/sctu/stripe/pay-server/foo.rb","languageId":"ruby","version":1,"text":"# frozen_string_literal: true\n# typed: true\n\n\n\nclass Opus::Foo; extend T::Sig\n  sig {returns(String)}\n  def uniquebar\n    \"bar\"\n  end\n\n  sig {returns(String)}\n  def baz\n    uniquebar + \"baz\"\n  end\nend\n"}}}
 Read: {"jsonrpc":"2.0","id":1,"method":"workspace/symbol","params":{"query":"uniquebar"}}
-Write: {"jsonrpc":"2.0","id":1,"requestMethod":"workspace/symbol","result":[{"name":"uniquebar","kind":6,"location":{"uri":"file:///Users/sctu/stripe/pay-server/foo.rb","range":{"start":{"line":7,"character":2},"end":{"line":7,"character":15}}},"containerName":"::Opus::Foo"}]}
+Write: {"jsonrpc":"2.0","id":1,"requestMethod":"workspace/symbol","result":[{"name":"::Opus::Foo#uniquebar","kind":6,"location":{"uri":"file:///Users/sctu/stripe/pay-server/foo.rb","range":{"start":{"line":7,"character":2},"end":{"line":7,"character":15}}},"containerName":"::Opus::Foo"}]}
```

Comment:
Hmm, can we somehow avoid having the `::` prefix? That explicitly means the "global scope" in Ruby, but I think it hurts readability (all of the results will be in the global scope).

Reply:
Updated to strip in 25be879d4